osu.Game/Overlays/BeatmapSet/Info.cs

```diff
@@ -119,6 +132,12 @@ public Info()
             {
                 source.Text = b.NewValue?.Metadata.Source ?? string.Empty;
                 tags.Text = b.NewValue?.Metadata.Tags ?? string.Empty;
+
+                var genreId = b.NewValue?.OnlineInfo.Genre ?? BeatmapSetOnlineGenre.Unspecified;
```

Comment:
why a local if it's used once

Reply:
Though it would be better than using .ToString() two times

Reply:
how is it using `ToString()` two times?

Reply:
Oh, yeah right, will fix